function_lab.js

```diff
@@ -0,0 +1,263 @@
+// # Function Exercises
```

Comment:
I see that you are still developing your skills on functions and scope. Reach out to me to go over this assignment and for any questions ! Im here for you ! :)